src/components/DetailsAndMap/Components/DataTabs.tsx

```diff
@@ -0,0 +1,381 @@
+`;
+
+const PersonDiv = styled.div`
+    display:flex;
+    justify-content:flex-start;
+    align-items:center;
+`;
+
+const PersonIconDiv = styled.div`
+    margin-top:13px;
+`;
+
+const PersonDetailDiv = styled.div`
+    margin-left:10px;
+`;
+
+const PersonName = styled.h5`
+    color:#696969;
+    font-weight:500;
+`;
+
+const Date = styled.p`
+    margin-top:-15px;
+    font-size:10px;
+    color:#696969;
+`;
+
+const CommentDiv = styled.div`
+    line-height:20px;
+    width:270px;
+    margin-left:5px;
+    margin-top:-10px;
+`;
+
+const Comment = styled.p`
+    color:#696969;
+    font-size:12px;
+    word-spacing:3px;
+`;
+
+
+// -----------------Review Data---------------
+
+const DIV = styled.div`
+display:flex;
+width:100%;
+`;
+
+const TabDataInfo = styled.div`
+    width:${(props) => props.theme.infoWidth};
+    transition:${(props) => props.theme.infoTransition && props.theme.infoTransition};
+    position:relative;
+    height:48vh;
+    display:flex;
+    flex-direction:column;
+    overflow:scroll;
+    box-sizing:border-box;
+
+`;
+
+const TabDataReview = styled.div`
+    width:${(props) => props.theme.reviewWidth};
+    transition:${(props) => props.theme.reviewTransition && props.theme.reviewTransition};
+    position:relative;
+    height:48vh;
+    display:flex;
+    flex-direction:column;
+    overflow:scroll;
+    box-sizing:border-box;
+`;
+
+const BorderDiv = styled.div`
+    position:relative;
+    width: 50%;
+    border-bottom: 3px solid #0097be;
+    margin-top: -3px;
+    left:${(props) => props.theme.left};
+    transition: left 200ms;
+`;
+
+const DataTabs: React.FC = () => {
+    const [infoActive, setInfoActive] = useState(true);
+
+    const info = {
+        infoWidth: "100%",
+        reviewWidth: "0",
+        reviewTransition: "width 200ms",
+        infoTransition: "width 200ms",
+        left: "0",
+    }
+
+    const review = {
+        infoWidth: "0",
+        reviewWidth: "100%",
+        infoTransition: "width 200ms",
+        reviewTransition: "width 200ms",
+        left: "50%",
+    }
+
+    return (
+        <ThemeProvider theme={infoActive ? info : review}>
+            <Container>
+                <TabContainer>
+                    <TabInfo onClick={() => setInfoActive(true)}>Info</TabInfo>
+                    <TabReview onClick={() => setInfoActive(false)}> Reviews</TabReview>
+                </TabContainer>
+                <BorderDiv />
+                <DIV>
+                    <TabDataInfo>
+                        <DataContainer>
+                            <HomeIconDiv><HomeIcon style={{ color: "#318CE7" }} /> </HomeIconDiv>
+                            <DetailDiv>
+                                <DetailHeading>Entire Apartment</DetailHeading>
+                                <DetailDesc>4 Guests 1 bedroom 4 Guests 1 bedroom</DetailDesc>
+                            </DetailDiv>
+                        </DataContainer>
+                        <DataContainer>
```

Comment:
please create reuseable component instead of repeating the code

Reply:
done